ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +650,110 @@ def patch_default_sc_to_non_default(self):
+                                 resource_name='cluster-monitoring-config')
+        monitoring_list = monitoring_obj.get('data').get('config.yaml')
+        for item in monitoring_list:
+            if item.get().get('volumeClaimTemplate').get('spec').get('storageClassName') in sc_list:
+                monitoring_list.remove(item)
+        monitoring_obj.patch(resource_name='cluster-monitoring-config',
+                             params=f'{{"data": {{"config.yaml": {monitoring_list}}}}}')
+
+        # Removing OpenShift Container Platform registry from OpenShift Container Storage
+        image_registry_obj = ocp.OCP(namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE)
+        if self.platform == constants.AWS_PLATFORM:
+            image_registry_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                                     params=" '{\"spec\":{\"storage\":{}}}' ")
+
+        elif self.platform == constants.VSPHERE_PLATFORM:
+            image_registry_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                                     params=" '{\"spec\":{\"storage\":{\"emptyDir\":{}}}}' ")
+
+        # Removing the cluster logging operator from OpenShift Container Storage
+        # Remove the ClusterLogging instance in the namespace.
+        ocp_obj.exec_oc_cmd("delete clusterlogging instance -n openshift-logging --wait=true --timeout=5m")
+
+        # STEP3: Follow these instructions to ensure that the PVCs listed in the previous step are deleted
+        # STEP 3.1: Determine the pod that is consuming the PVC.
+        pod_list = get_all_pods()
+        for pod in pod_list:
+            for pvc in pvcs_to_delete:
+                if get_pvc_name(pod) == pvc.get('metadata').get('name'):
+                    pod.delete()
+
+        # STEP 3.2: Delete the PVCs.
+        for pvc in pvc_list:
+            pvc.delete()
+            ocp_obj.wait_for_delete(pvc.name)
+
+        # STEP 4: Delete the StorageCluster object.
+        storage_cluster = ocp.OCP(kind=constants.STORAGECLUSTER, resource_name="ocs-storagecluster")
+        storage_cluster.delete()
+
+        # STEP 5: Delete the namespaces and wait till the deletion is complete.
+        ocp_obj.delete_project('openshift-storge')
+        ocp_obj.wait_for_delete("openshift-storage")
+
+        # STEP 6: List the storage nodes.
+        nodes_list = get_labeled_nodes('cluster.ocs.openshift.io/openshift-storage')
+
+        # STEP 7: For each node, perform the following:
+        for node in nodes_list:
+            ocp_obj.exec_oc_debug_cmd(node=node, cmd_list=["rm -rf /var/lib/rook"])
+
+        # STEP 8: Delete the storage classes with an openshift-storage provisioner listed in step 1.
+        for storage_class in sc_list:
+            storage_class.delete()
+
+        # STEP 9: Remove the taint from the storage nodes.NOT SUPPORTED YET
+        """ ocp_obj.exec_oc_cmd("adm taint nodes --all node.ocs.openshift.io/storage-") """
```

Comment:
this is in a comment - what is the plan?

Reply:
After talking to Elad about how to change it from `exac_oc_cmd` to something better, he said that the taints are not yep implemented and this step is not relevant for now. Decided to leave it commented until it will be relevant

Reply:
ok. Please put it in a proper way:
```python
# TODO: add taints when needed, i.e:
# ocp_obj.exec_oc_cmd("adm taint nodes --all node.ocs.openshift.io/storage-")
```